### ISSUE TITLE
What is the train details on ImageNet2012?

### ISSUE THREAD

**Issue by qingzhouzhen** [sanghoon/pva-faster-rcnn#93]
I want to pretrain pvanet on ImageNet, but can not find the details arguments such opotimizer, how to initilize weight and so on.(except learning rate and 192*192 I found in article) @sanghoon

**Comment by twmht** [comment]
@sanghoon 

Can you share your plateau window size when training imageNet?

**Comment by qingzhouzhen** [comment]
what your problem? @twmht , my problem is I pretrain pvanet on ImageNet2012 got 66%, not 70%, I want to transfer pvanet to MXNet.

**Comment by twmht** [comment]
@qingzhouzhen 

I go 61%, but the learning rate policy is `step`, not `plateau`.

Since @sanghoon  does not provide the solver to train imageNet, I don't know how to set the plateau parameter to reach 2M iterations.

**Comment by twmht** [comment]
@qingzhouzhen 

I am curious about what's your learning rate policy?

**Comment by qingzhouzhen** [comment]
--lr-factor=0.36  --lr-step-epoch=30,60,80,100,120,140  --num-epochs=160
by the way, what is your framework? Caffe , TensorFlow, or Mxnet

**Comment by twmht** [comment]
@qingzhouzhen 

I am using caffe, so you do not use the learning rate policy `plateau`?

**Comment by qingzhouzhen** [comment]
I do not no what is plateau, MXNET does not have this , at least I do not know.
